src/hooks/index.ts

```diff
@@ -111,6 +104,22 @@ export function useEagerConnect() {
             setTried(true)
           }
         })
+      } else if (window.xfi && window.xfi.ethereum) {
+        try {
+          await activate(xDefi, undefined, true)
+          setTried(true)
+        } catch (error) {
+          setWallet(null)
+          setTried(true)
+        }
+      } else if (window.bitkeep && window.isBitKeep) {
+        try {
+          await activate(bitKeep, undefined, true)
+          setTried(true)
+        } catch (error) {
+          setWallet(null)
+          setTried(true)
+        }
```

Comment:
we don't need to add these condition here. its already handled n 95-106 line number

Reply:
@prodesert22 may be we can change  95-106 line number to 
```
if (connector?.activate) {
  activate(connector, undefined, true).catch(() => {
    setWallet(null)
    setTried(true)
  })
}
```
so it will work for every connector right ? and connector we decide based on last wallet user has connected. what do you think ?

Reply:
I fixed this, I change the useMemo on line 35, it is returning `window.ethereum` but on custom connector we need to check window.xfi or window.bitkeep if it exist, return xdefi or bitkeep connector